integration_test/test_videoroom/test/support/simulcast_mustang.ex

```diff
@@ -0,0 +1,92 @@
+defmodule SimulcastMustang do
+  use Stampede.Mustang
+
+  @impl true
+  def join(browser, options) do
+    page = browser |> Playwright.Browser.new_page()
+    _response = Playwright.Page.goto(page, options.target_url)
+
+    Process.sleep(500)
+
+    :ok = Playwright.Page.click(page, "[id=#{options.start_button}]")
+
+    {browser, page}
+  end
+
+  @impl true
+  def afterJoin({browser, page}, options) do
+    Process.sleep(options.join_interval)
+
+    {browser, page}
+  end
+
+  @impl true
+  def linger({_browser, page} = ctx, options) do
```

Comment:
Let's try to make this more generic by removing `when button in [...]` guard and adding `get_stats` action

After this we can try to use it in `stampede_test.exs`.